docs/images/bazzite/installation.md

```diff
@@ -72,8 +72,8 @@ If you need any additional applications check out the software center that comes
 
 ## Gaming Tips and Tricks
 
-For a quick runthrough of the basics of gaming on desktop Linux take a look [here.](https://github.com/ublue-os/website/new/main/docs/images/bazzite/gaming_guide.md)
+For a quick runthrough of the basics of gaming on desktop Linux take a look [here.](https://github.com/ublue-os/website/blob/main/docs/images/bazzite/gaming_guide.md)
 
 ## Waydroid
 
-If you wish to use Waydroid, a setup guide can be found [here](/images/bazzite/waydroid).
+If you wish to use Waydroid, a setup guide can be found [here](https://universal-blue.org/images/bazzite/waydroid/).
```

Comment:
```suggestion
```

mkdocs does this for you so this one is set, you only need the stuff after the domain and no .md